### DISCUSSION TITLE
DOI link to latest version

### DISCUSSION THREAD

**Discussion by nohcnamp** [sertit/eoreader#73] [Q&A] [answered]
Hello Rémi,

I noticed that you updated the DOI badge by hand. Did you know that https://doi.org/10.5281/zenodo.5082050 always redirects to the latest version ?

Respectfully
Pierre

**Comment by remi-braun** [comment, marked as answer]
Hey Pierre,

I was looking for it a long time ago, without success obvisously 😄 
I will update it tomorrow, thanks a lot!

**Reply by nohcnamp** [reply]
Glad I could help ! :smile:
